vision_datasets/common/data_manifest.py

```diff
@@ -95,15 +99,35 @@ def __init__(self, id, img_path, width, height, labels):
             labels (list or dict):
                 classification: [c_id] for multiclass, [c_id1, c_id2, ...] for multilabel;
                 detection: [[c_id, left, top, right, bottom], ...];
-                image caption: [caption1, caption2, ...];
-                image_text_matching: [(text1, match (0 or 1), text2, match (0 or 1), ...)]
+                image_caption: [caption1, caption2, ...];
+                image_text_matching: [(text1, match (0 or 1), text2, match (0 or 1), ...)];
                 multitask: dict[task, labels]
+                image_matting: [mating1, matting2, ...]
```

Comment:
Per @shonohs's  old comment, what is matting?

Reply:
changed to: "image_matting: [mask1, mask2, ...], each mask is a 2D numpy array that has the same width and height with the image."